capstone/src/main/webapp/details.js

```diff
@@ -107,3 +110,19 @@ function appendCuisineTag(cuisineName) {
         .appendChild(outerDiv);
   }
 }
+
+/**
+ * Update the page views for a restaurant using the PageViewServlet.
+ * @param restaurantKey
+ * @param restaurantName
+ * @return {Promise<void>}
+ */
+async function updatePageViews(restaurantKey, restaurantName) {
```

Comment:
Mentioned this in a previous review, but not sure where it went: Why is restaurant name needed here? The key uniquely identifies the restaurant.

Reply:
Removed the name param!